src/arrays/programs_involving_multidimensional_data/medium/q2/MultiplyArrays.java

```diff
@@ -0,0 +1,49 @@
+package arrays.programs_involving_multidimensional_data.medium.q2;
+
+/*
+Write a Java program that initializes a 2 x 3 2D array with values {{2, 3, 4}, {4, 5, 1}}. Next, the program needs to
+randomly generate an integer multiplier in the range of 5 and 10 (inclusive of both values). The program prints the
+multiplier then calls a method increaseBy that takes the 2D array and the multiplier as input, multiplies all the
+elements in the 2D array by the multiplier, and returns nothing as output. Lastly, the program displays all the elements
+in the array after the multiplier is applied. Note that you increaseBy method should be generalized to work with any 2D
+array length.
+
+Sample output:
+    8
```

Comment:
We should add some print statements so it is not just numbers in the output. Maybe something like
```
Random Multiplier: 8
Output Array: 
1 2 3
4 5 6
```